src/html5/tokenizer/token.rs

```diff
@@ -19,37 +19,32 @@ pub struct Attribute {
 }
 
 /// The different token structures that can be emitted by the tokenizer
-#[derive(Clone, PartialEq)]
+#[derive(Clone, Debug, PartialEq)]
 pub enum Token {
-    DocTypeToken {
+    DocType {
         name: Option<String>,
         force_quirks: bool,
         pub_identifier: Option<String>,
         sys_identifier: Option<String>,
     },
-    StartTagToken {
+    StartTag {
         name: String,
         is_self_closing: bool,
         attributes: HashMap<String, String>,
     },
-    EndTagToken {
+    EndTag {
         name: String,
         is_self_closing: bool,
-        attributes: HashMap<String, String>,
```

Comment:
I don't think `attributes` is needed.  I suspect `is_self_closing` is also not needed, but I did not look into this question.

Reply:
in theory you could do:

```
</foo bar="baz" />
```

although i doubt this would actually result in anything.